config-linux.md

```diff
@@ -264,8 +264,8 @@ Values for memory specify the limit in bytes, or `-1` for unlimited memory.
 * **`limit`** *(int64, OPTIONAL)* - sets limit of memory usage
 * **`reservation`** *(int64, OPTIONAL)* - sets soft limit of memory usage
 * **`swap`** *(int64, OPTIONAL)* - sets limit of memory+Swap usage
-* **`kernel`** *(int64, OPTIONAL)* - sets hard limit for kernel memory
-* **`kernelTCP`** *(int64, OPTIONAL)* - sets hard limit for kernel TCP buffer memory
+* **`kernel`** *(int64, OPTIONAL, NOT RECOMMENDED)* - sets hard limit for kernel memory
+* **`kernelTCP`** *(int64, OPTIONAL, NOT RECOMMENDED)* - sets hard limit for kernel TCP buffer memory
```

Comment:
Wondering if it would be better to mark them as "deprecated" (as they're deprecated  in the Kernel); kept for backward compatibility by runtimes are allowed to ignore them

Reply:
1. We're using the terms (OPTIONAL, NOT RECOMMENDED) as per RFC2119, and there's no such term as "DEPRECATED" in its lexicon.
2. This spec (if I understand it right) is for both runtimes and their users. Saying "OPTIONAL" means users can chose to not set these limits, and runtimes can ignore the set limits. Saying NOT RECOMMENDED means users and runtimes should not set those limits, unless there's a good understanding and some compelling reasons to do so.